content/en/basics/project-and-community/the-filecoin-project/index.md

```diff
@@ -24,8 +24,9 @@ The [Filecoin Community Roadmap](https://github.com/filecoin-project/community/d
 
 ## Research
 
-Learn about the ongoing cryptography research and design efforts that underpin the Filecoin protocol on the [Filecoin Research website](https://research.filecoin.io/). The [CryptoLab at Protocol Labs](https://research.protocol.ai/groups/cryptolab/) also actively researches improvements.
+Learn about the ongoing cryptography research and design efforts that underpin the Filecoin protocol on the [Filecoin Research website](https://github.com/filecoin-project/research). The [CryptoLab at Protocol Labs](https://research.protocol.ai/groups/cryptolab/) also actively researches improvements.
```

Comment:
Link detector flagged https://research.filecoin.io/. When I hit it in the browser, I see the following error.

research.filecoin.io’s DNS address could not be found. Diagnosing the problem.
DNS_PROBE_POSSIBLE

Changed it to https://github.com/filecoin-project/research. Is this correct?

@johnnymatthews